configs/ote/custom-object-detection/gen3_mobilenetV2_SSD/model.py

```diff
@@ -18,6 +18,7 @@
         anchor_generator=dict(
             type='SSDAnchorGeneratorClustered',
             strides=(16, 32),
+            reclustering_anchors=True,
```

Comment:
@druzhkov-paul I know, you consider this change as weird, because it doesn't impact the anchor generator itself, so it's just the way to place the parameter used before training. But for me, it's a convenient place, because:
1. In config we can switch on/off clustering
2. This parameter impacts changing the widths and heights of anchors, so it's convenient that all of them are close to each other
3. This parameter relates to the anchor generator, so placing it in another field could be confusing.
4. I also rely on the practice of adaptive dataset parameter, which was placed in the `RepeatDataset` field, although it was just a way to pass the parameter to the training utility. 

But I agree that it is strange to update the code of anchor generator, and if I didn't convince you, there are 2 options to replace it:
1. Put it in another place of the config outside the `model` field 
2. Put it in the template, as it is the part of the preparation on the Task side - but this parameter is not supposed to be changed, so it's better to keep it in model.py

Reply:
I'm still not happy with this design, and hope we (you) will have to change it moving forward 😄 . Alternative option №1 looks better to me. But let's merge it as is for now, while tests are passing.